app/components/chart-options-selector.tsx

```diff
@@ -91,12 +99,12 @@ const ActiveFieldSwitch = ({
   );
 
   // It's a dimension which is not mapped to an encoding field, so we show the filter!
-  // FIXME: activeField and encodingField should match? remove type assertion
+  // FIXME: activeField and encodingField should match? to remove type assertion
   if (
     !encodings.map((e) => e.field).includes(activeField as EncodingField) &&
```

Comment:
Maybe simplify this to `encodings.find(e => e.field === activeField)` to save a temporary array allocation.